docs/user_guide/tutorials/intro.md

```diff
@@ -15,7 +15,7 @@ By the end of this tutorial, you will:
 
 :::{admonition} Prerequisites
 :class: warning
-Before beginning this tutorial, JupyterGIS must be installed on your computer (see [Installation instructions](https://geojupyter.github.io/jupytergis/install.html)) or you can use an online version of JupyterGIS (such as [![Jupyterlite badge](https://jupyterlite.rtfd.io/en/latest/_static/badge.svg)](https://geojupyter.github.io/jupytergis/lite/lab/index.html?path=france_hiking.jGIS/)).
+Before beginning this tutorial, JupyterGIS must be installed on your computer (see [Installation instructions](https://jupytergis.readthedocs.io/en/latest/user_guide/install.html)) or you can use an online version of JupyterGIS (such as [![Jupyterlite badge](https://jupyterlite.rtfd.io/en/latest/_static/badge.svg)](https://geojupyter.github.io/jupytergis/lite/lab/index.html?path=france_hiking.jGIS/)).
```

Comment:
Should this be an internal link instead of a full HTTP URL so that the check links job can catch this type of mistake?

Reply:
I'm not sure actually, I don't have a strong opinion on it.

Thankfully check-links allows us to catch this issue early anyway. And those links are not likely to change every month. So I guess it's not a big deal?

Reply:
I'm not sure I understand! Just to be really clear, I'm suggesting:
```suggestion
Before beginning this tutorial, JupyterGIS must be installed on your computer (see [Installation instructions](../install.html)) or you can use an online version of JupyterGIS (such as [![Jupyterlite badge](https://jupyterlite.rtfd.io/en/latest/_static/badge.svg)](https://geojupyter.github.io/jupytergis/lite/lab/index.html?path=france_hiking.jGIS/)).
```

Then if we move documents around in a PR (like #411), the check-links check for that PR would fail.

Reply:
Yep, got it 👍🏽 But the nice thing about keeping URLs to the docs is that when you click on it from the repo you get directed to the docs deployment.

Reply:
Are there other links in the docs that are full HTTPs urls? As far as I could tell, we're using relative links everywhere else. The thing I'm confused about most right now is why this one is different. Since we have so much syntax in the docs that GitHub can't understand, I assumed it was a non-goal to serve folks who are viewing the docs in GitHub!